frontend/device/android/device.lua

```diff
@@ -246,6 +246,11 @@ function Device:init()
         end,
     }
 
+    -- disable translation map for specific models
+    if android.prop.model == "xiaomi_reader" or android.prop.model == "moaanmix7" then
```

Comment:
While it's so limited in scope a table would be overengineering, but I'd still make it alphabetic unless there's a reason not to.

Reply:
Alphabetic makes sense. Updated.